### PR TITLE
LCH-6073: Add command base and pq:all command.

### PR DESCRIPTION
**Pull request by saxumVermes**

### CODE REVIEW

File: src/Command/Helpers/ColorizedOutputTrait.php [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+   */
+  protected $options = 'options=bold';
+
+  /**
+   * Sets the output for the outputted text.
+   *
+   * @param string $options
+   *   Set the options for the output.
+   */
+  public function setOutputOptions(string $options) {
```
Comment (narendradesai) [inline src/Command/Helpers/ColorizedOutputTrait.php:23]: Please add return type hint `void`

File: tests/Command/PqCommands/ContentHubPqBundleTest.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+      ->willReturn($this->prophesize(HelperSet::class)->reveal());
+
+    $this->command = new ContentHubPqBundle();
+    $this->command->setApplication($application->reveal());
+  }
+
+  /**
+   * Tests --exclude option.
+   */
+  public function testGetPqCommandsWithExcludedCommands() {
```
Comment (narendradesai) [inline tests/Command/PqCommands/ContentHubPqBundleTest.php:62]: Return type hint missing.

File: tests/Command/PqCommands/ContentHubPqBundleTest.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    $this->assertEquals([], $list);
+
+    $list = $this->command->getPqCommands([], 'exclude');
+    $this->assertEquals(array_values($this->testCommands), $list);
+  }
+
+  /**
+   * Tests --checks option.
+   */
+  public function testGetPqCommandsWithIncludedCommands() {
```
Comment (narendradesai) [inline tests/Command/PqCommands/ContentHubPqBundleTest.php:82]: Return type hint missing.

File: tests/Command/PqCommands/ContentHubPqBundleTest.php [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+    $list = $this->command->getPqCommands([], 'checks');
+    $this->assertEquals([], $list);
+  }
+
+  /**
+   * Tests options when --checks and --exclude are both provided.
+   *
+   * @covers ::getCommandListAndFilterMode
+   */
+  public function testExecuteWithExcludeAndInclude() {
```
Comment (narendradesai) [inline tests/Command/PqCommands/ContentHubPqBundleTest.php:104]: Return type hint missing.

**Review by narendradesai — CHANGES_REQUESTED** [review]

**Review by narendradesai — APPROVED** [review]
LGTM 👍🏻

**Review by amangrover90 — APPROVED** [review]
Approach looks good. Only question is should we be merging this to main branch right now or  merging it in a pq-feature branch until whole functionality is complete.
We've to release all CHUC libraries to drop symfony 3.4 support next sprint.
Thoughts @saxumVermes ?

**Comment by saxumVermes** [comment]
Good point, easier to get our head around it! Closing this PR!

**Pull request closed without merging**